schemas/IDLs/org.gel.models.report.avro/6.0.0/ExitQuestionnaire.avdl

```diff
@@ -0,0 +1,360 @@
+        */
+        // TODO: use VariantCoordinates in CommonInterpreted.avdl
+        string variantDetails;
+
+        /**
+        How has/will this potentially actionable variant been/be used?
+        */
+        CancerUsabilityGermline variantUsability;
+
+        /**
+        Has this variant been tested by another method (either prior to or following receipt of this WGA)?
+        */
+        CancerTested variantTested;
+
+        /**
+        Please enter validation assay type e.g Pyrosequencing, NGS panel, COBAS, Sanger sequencing. If not applicable enter NA;
+        */
+        string validationAssayType;
+    }
+
+    /**
+    The questions for the cancer program exit questionnaire for somatic variants
+    */
+    record CancerSomaticVariantLevelQuestions{
+        /**
+        Variant coordinates following format `Chr: Pos Ref > Alt`
+        */
+        // TODO: use VariantCoordinates in CommonInterpreted.avdl
+        string variantDetails;
+
+        /**
+        Type of potential actionability:
+        */
+        CancerActionability variantActionability;
+
+        /**
+        Other information about variant actionability
+        */
+        union {null, string} otherVariantActionability;
+
+        /**
+        How has/will this potentially actionable variant been/be used?
+        */
+        CancerUsabilitySomatic variantUsability;
+
+        /**
+        Has this variant been tested by another method (either prior to or following receipt of this WGA)?
+        */
+        CancerTested variantTested;
+
+        /**
+        Please enter validation assay type e.g Pyrosequencing, NGS panel, COBAS, Sanger sequencing. If not applicable enter NA;
+        */
+        string validationAssayType;
+    }
+
+    /**
+    The cancer program exit questionnaire
+    */
+    record CancerExitQuestionnaire{
+        /**
+        The date when the questionnaire was submitted
+        */
+        string eventDate;
+
+        /**
+        The person that submitted the questionnaire
+        */
+        string reporter;
+
+        /**
+        The case level questions
+        */
+        CancerCaseLevelQuestions caseLevelQuestions;
+
+        /**
+        The questions for somatic variants
+        */
+        union {null, array<CancerSomaticVariantLevelQuestions>} somaticVariantLevelQuestions;
+
+        /**
+        The questions for germline variants
+        */
+        union {null, array<CancerGermlineVariantLevelQuestions>} germlineVariantLevelQuestions;
+
+        /**
+        Please enter any additional comments you may have about the case here.
+        */
+        union {null, string} additionalComments;
+
+        /**
+        Other actionable variants or entities.
+        Please provide other (potentially) actionable entities: e.g domain 3 small variants,
+        SV/CNV, mutational signatures, mutational burden
+        */
+        union {null, string} otherActionableVariants;
```

Comment:
in 5 this was:
```
union {null, array<AdditionalVariantsQuestions>} otherActionableVariants;
```
are we not likely to get some nonsense data is this is a string ?